docs/en/04-supported-codebases/mmcls.md

```diff
@@ -1,20 +1,163 @@
+
+The command below shows an example about converting `resnet18` model to onnx model that can be inferred by ONNX Runtime.
+
+```shell
+cd mmdeploy
+
+# download resnet18 model from mmcls model zoo
+mim download mmcls --config resnet18_8xb32_in1k --dest .
+
+# convert mmcls model to onnxruntime model with dynamic shape
+python tools/deploy.py \
+    configs/mmcls/classification_onnxruntime_dynamic.py \
+    resnet18_8xb32_in1k.py \
+    resnet18_8xb32_in1k_20210831-fbbb1da6.pth \
+    tests/data/tiger.jpeg \
+    --work-dir mmdeploy_models/mmcls/ort \
+    --device cpu \
+    --show \
+    --dump-info
+```
+
+It is crucial to specify the correct deployment config during model conversion. We've already provided builtin deployment config [files](https://github.com/open-mmlab/mmdeploy/tree/dev-1.x/configs/mmcls) of all supported backends for mmclassification. The config filename pattern is:
+
+```
+classification_{backend}-{precision}_{static | dynamic}_{shape}.py
+```
+
+- **{backend}:** inference backend, such as onnxruntime, tensorrt, pplnn, ncnn, openvino, coreml and etc.
+- **{precision}:** fp16, int8. When it's empty, it means fp32
+- **{static | dynamic}:** static shape or dynamic shape
+- **{shape}:** input shape or shape range of a model
+
+Therefore, in the above example, you can also convert `resnet18` to other backend models by changing the deployment config file `classification_onnxruntime_dynamic.py` to [others](https://github.com/open-mmlab/mmdeploy/tree/dev-1.x/configs/mmcls), e.g., converting to tensorrt-fp16 model by `classification_tensorrt-fp16_dynamic-224x224-224x224.py`.
+
+```{tip}
+When converting mmcls models to tensorrt models, --device should be set to "cuda"
+```
+
+## Model Specification
+
+Before moving on to model inference chapter, let's know more about the converted model structure which is very important for model inference.
+
+The converted model locates in the working directory like `mmdeploy_models/mmcls/ort` in the previous example. It includes:
+
+```
+mmdeploy_models/mmcls/ort
+├── deploy.json
+├── detail.json
+├── end2end.onnx
+└── pipeline.json
+```
+
+in which,
+
+- **end2end.onnx**: backend model which can be inferred by ONNX Runtime
+- **deploy.json**: meta information about backend model
+- **pipeline.json**: inference pipeline of mmdeploy SDK
+- **detail.json**: conversion parameters
+
+The whole package **mmdeploy_models/mmcls/ort** is defined as **mmdeploy SDK model**, i.e., **mmdeploy SDK model** includes both backend model and inference meta information.
+
+## Model inference
+
+### Backend model inference
+
+MMDeploy provides a unified API named as `inference_model` to inference model, making all inference backends API transparent to users.
+
+Take the previous converted `end2end.onnx` model as an example, you can use the following code to inference the model.
+
+```shell
+from mmdeploy.apis import inference_model
+result = inference_model(
```

Comment:
Should we use task processor instead of inference model api?

Reply:
Sure